### PR TITLE
Update I18n.json

### PR DESCRIPTION
**Pull request by dmitrox3**
переклад на російську

### CODE REVIEW

File: src/Mi899.Data/I18n.json [hunk 1]
```diff
@@ -454,4 +454,70 @@
...
+    "BiosesPartialForm.grdBioses.colName.HeaderText": "Название",
+    "BiosesPartialForm.grdBioses.colProperties.HeaderText": "Свойства",
+    "BiosesPartialForm.grdBioses.colTags.HeaderText": "Теги",
+    "BiosesPartialForm.lblSearch.Text": "Поиск:",
+    "MainForm.msiExit.Text": "Выход",
+    "MainForm.msiExplore.Text": "Осмотреть",
+    "MainForm.msiExploreBioses.Text": "BIOS",
+    "MainForm.msiExploreMotherboards.Text": "Материнские платы",
+    "MainForm.msiFile.Text": "Файл",
+    "MainForm.msiFileReadMe.Text": "Прочитай меня!",
```
Comment (dmitrox3) [inline src/Mi899.Data/I18n.json:474]: add Russian translate

File: src/Mi899.Data/I18n.json [hunk 2]
```diff
@@ -454,4 +454,70 @@
...
+    "MainForm.msiHelp.Text": "Помощь",
```
Comment (dmitrox3) [inline src/Mi899.Data/I18n.json]: add Russian translate

**Comment by dmitrox3** [comment]
add Russian translate

**Comment by miyconst** [comment]
@dmitrox3 thank you for the contribution attempt. The Russian translation has been added in the [`1.0.2`](https://github.com/miyconst/Mi899/releases/tag/1.0.2) version.

**Pull request closed without merging**